tests/MoverGroupper.test.tsx

```diff
@@ -969,4 +969,115 @@ describe("MoverGroupper", () => {
                 );
             });
     });
+
+    it("should handle tabbing in groupper/mover/groupper when the inner groupper container is delegated and not focusable", async () => {
+        await new BroTest.BroTest(
+            (
+                <div {...getTabsterAttribute({ root: {} })}>
+                    <button>Button1</button>
+                    <div
+                        id="outer-groupper"
+                        tabIndex={0}
+                        {...getTabsterAttribute({
+                            groupper: {
+                                tabbability:
+                                    Types.GroupperTabbabilities
+                                        .LimitedTrapFocus,
+                            },
+                        })}
+                    >
+                        <button>Button2</button>
+                        <div
+                            {...getTabsterAttribute({
+                                mover: {
+                                    visibilityAware:
+                                        Types.Visibilities.PartiallyVisible,
+                                },
+                            })}
+                        >
+                            <button>Button3</button>
+                            <div
```

Comment:
how does down arrow work for a delegated groupper while in a mover ?

Reply:
Cool, can we can we test that as well ?

Reply:
I believe we do test it in various tests involving grouppers. This particular test and the test above it test it too.